src/Sarif.WorkItemFiling/GitHub/GitHubFilingTarget.cs

```diff
@@ -4,6 +4,7 @@
 using System;
 using System.Collections.Generic;
 using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Sarif.WorkItemFiling.Grouping;
```

Comment:
>WorkItemFiling.Grouping [](start = 35, length = 23)

Here's a code smell I'm always skeptical of: no code changed in this file and you were required to add a namespace.

Reply:
This isn't a code smell. When I move a type from one namespace to another, client code changes. Once the type is in the right namespace, this doesn't happen again. This change is just "I made a mistake, I put a type in the wrong namespace".

---
In reply to: [306119188](https://github.com/microsoft/sarif-sdk/pull/1582#discussion_r306119188) [](ancestors = 306119188)

Reply:
Gotcha.

---
In reply to: [306310265](https://github.com/microsoft/sarif-sdk/pull/1582#discussion_r306310265) [](ancestors = 306310265,306119188)